code/datums/components/uplink.dm

```diff
@@ -128,6 +129,15 @@
 	if(istype(item, /obj/item/stack/telecrystal))
 		load_tc(user, item)
 
+/datum/component/uplink/proc/on_examine(datum/souuce, mob/user, list/examine_list)
```

Comment:
there's a small typo in the name of the "datum/source" variable

Reply:
well its not like the vars used anyway

Reply:
C'mon